manifests/operators/prometheus-bosh-exporter.yml

```diff
@@ -0,0 +1,65 @@
+# Prometheus Scrape Config
+- type: replace
+  path: /instance_groups/name=prometheus/jobs/name=prometheus/properties/prometheus/scrape_configs/-
+  value: 
+    job_name: bosh
+    static_configs:
+    - targets: 
+      - localhost:9190 # bosh_exporter
+
+# Exporter jobs
+- type: replace
+  path: /instance_groups/name=prometheus/jobs/-
+  value: 
+    name: bosh_exporter
+    release: prometheus
+    properties:
+      bosh_exporter:
+        metrics:
+          environment: ((metrics_environment))
+        bosh:
+          url: ((bosh_url))
+          username: ((bosh_username))
+          password: ((bosh_password))
+          ca_cert: ((bosh_ca_cert)
+
+# Grafana Dashboards
+- type: replace
```

Comment:
These dashboards are only useful if you have deployed the `node-exporter` addon, so better to add them on a `node-exporter` operator.